layouts/partials/header.html

```diff
@@ -1,4 +1,8 @@
-<header class="header">
+{{ $extra_header_class := "" }}
```

Comment:
Wonder if this could just be a one-liner
<header class="header{{ if .Site.Params.sticky_header }} sticky{{ end }}">